tools/rust_api/src/database.rs

```diff
@@ -46,6 +50,8 @@ impl Default for SystemConfig {
             max_num_threads: 0,
             enable_compression: true,
             read_only: false,
+            // This is a little weird, but it's a temporary interface
+            max_db_size: u32::MAX as u64,
```

Comment:
Why not initialize `max_db_size` in a similar way to `buffer_pool_size`?

Reply:
I was trying to match the default argument for the constructor, but it guess the rust API isn't using the C++ SystemConfig constructor anyway (it probably should, but I think it was written when the constructor was different).

On the other hand, I think both bufferPoolSize should default to zero instead since `-1u` is a valid value that just gets turned into auto-detection.